cmd/power/power.go

```diff
@@ -197,7 +230,7 @@ func shutDownContainers() error {
 		}
 
 		log.Infof("Stopping %s : %v", container.ID[:12], container.Names)
-		stopErr := client.StopContainer(container.ID, uint(timeout))
+		stopErr := client.StopContainer(container.ID, 0)
```

Comment:
Don't put `0` here.  That means docker will just `SIGKILL` all the containers.  We need some timeout so that containers have an option to gracefully shutdown.  I think 2 or 5 seconds should be good.